libs/community/langchain_community/llms/mlrun.py

```diff
@@ -0,0 +1,170 @@
+from collections.abc import AsyncIterator, Iterator
+from typing import Any, List, Optional, Union
+
+import mlrun
+from langchain_core.language_models.base import LanguageModelInput
+from langchain_core.language_models.llms import LLM
+from langchain_core.pydantic_v1 import Field
+from langchain_core.runnables import RunnableConfig
+
+
+class Mlrun(LLM):
+    """A class for interacting with Langchain, while using models served by mlrun."""
+
+    llm: Any = Field(default=None)
+    name: str = Field(default=None)
+
+    def __init__(
+        self,
+        nuclio_function: Any,
+        name: str,
+        **kwargs: dict,
+    ):
+        super().__init__(**kwargs)
+        self.llm = nuclio_function
+        self.name = name
+
+    @property
+    def _llm_type(self) -> str:
+        """Get the type of language model used by this chat model.
+        Used for logging purposes only."""
+        return "Mlrun"
+
+    def _call(
+        self,
+        prompt: str,
+        **generate_kwargs: dict,
+    ) -> str:
+        pass
```

Comment:
I guess you put it here because of an abstraction, but leave a comment why you didn't implement it.